docs/core/docker/container-images.md

```diff
@@ -0,0 +1,126 @@
+---
+title: .NET container images
+description: Learn about the various .NET container images available on Docker Hub.
+ms.date: 09/22/2023
+---
+
+# .NET container images
+
+.NET provides various container images for different scenarios. This article describes the different types of images and how they're used. For more information about official images, see the [Docker Hub: Microsoft .NET](https://hub.docker.com/_/microsoft-dotnet) repository.
+
+## Tagging scheme
+
+Starting with .NET 8, container images are more pragmatic in how they're differentiated. The following characteristics are used to differentiate images:
+
+- The target framework moniker (TFM) of the app.
+- The OS, version and architecture.
+- The image type (for example, `runtime`, `aspnet`, `sdk`).
+- The image variant (for example, `*-distroless`, `*-chiseled`).
+- The image feature (for example, `*-aot`, `*-extra`).
+
+## Images optimized for size
+
+The following images are focused on resulting in the smallest possible image size:
+
+- Alpine
+- Mariner distroless
+- Ubuntu chiseled
+
+These images are smaller, as they don't include globalization dependencies such as, ICU or tzdata. These images only work with apps that are configured for globalization invariant mode. To configure an app for invariant globalization, add the following property to the project file:
+
+```xml
+<PropertyGroup>
+  <InvariantGlobalization>true</InvariantGlobalization>
+</PropertyGroup>
+```
+
+> [!TIP]
+> SDK images aren't produced for `*-distroless` or `*-chiseled` image types. Composite images are the smallest `aspnet` offering for CoreCLR.
```

Comment:
Do we expect these customers to know what CoreCLR means?

Reply:
That's a good question. I'll define it before it's used.